internal/eval/evalers.go

```diff
@@ -973,11 +974,6 @@ func newInEval(lhs, rhs Evaler) Evaler {
 	return &inEval{lhs: lhs, rhs: rhs}
 }
 
-func hasKnown(known map[types.EntityUID]struct{}, k types.EntityUID) bool {
-	_, ok := known[k]
-	return ok
-}
-
 func entityInOne(env *Env, entity types.EntityUID, parent types.EntityUID) bool {
 	key := inKey{a: entity, b: parent}
```

Comment:
I thought the benchmarks indicated the cache wasn't providing much value?

Fine to leave changing the cache to another PR though.

Reply:
Yeah, I was going to do that in a separate PR.